examples/links/ece_export_for_twinbuilder.py

```diff
@@ -359,9 +518,14 @@ def read_parameters(json_file):
 file_id.close()
 
 # %%
-# Write the SML file.
+# Write the SML file
+# ~~~~~~~~~~~~~~~~~~
+# Generate the SML that will be loaded into Ansys Twin Builder to generate the ECE model. The SML
+# file uses the phase resistance and end winding inductance and data from the look-up table. The
+# SML file is saved using the path and filename taken from the ``ece_config.json`` configuration
+# file.
 file_id = open(sml_file, "w")
-file_id.write("%6s\r\n" % "MODELDEF ECER_Model1")
+file_id.write("%6s\r\n" % "MODELDEF ECE_e8")
```

Comment:
Could we get this from the mot file name, instead of hard coding? (Will need to be careful of non-alphanumeric characters in the mot file name, as these may not be valid in the SML, for example spaces, special characters or non-ascii characters)